src/routes/PlayQuiz/QuestionsList.tsx

```diff
@@ -41,6 +50,20 @@ export default function QuestionsList({
     }
   }
 
+  function getBadgeColor(points) {
+    if (parseInt(points) > minQuestionPoint + questionPointsAverage * 0.8) {
+      return 'red';
+    } else if (parseInt(points) > minQuestionPoint + questionPointsAverage * 0.6) {
+      return 'orange';
```

Comment:
Can we use colors from variables.css?

Reply:
we can use only those colors which are specified here
https://mantine.dev/core/badge/

Reply:
OK